README.md

```diff
@@ -387,6 +387,16 @@ This will record each token issued in your database, confirm it is still valid o
 
 For more in-depth documentation please see the [GuardianDb README](https://github.com/ueberauth/guardian_db/blob/master/README.md)
 
+## Best testing practises
+
+### How to add the token to a request (the Phoenix way)
+```elixir
+{:ok, token, _} = encode_and_sign(resource, %{}, token_type: :access)
```

Comment:
I would recommend to add a full example of the Controller so newbies do not get lost in the context.

For example, where `encode_and_sign/2`  come from?
Where should I add this code?